File: QtUi/window_editor.cpp
#include "window_editor.h"
#include "ui_window_editor.h"

window_editor::window_editor(QWidget *parent) :
    QMainWindow(parent),
    ui(new Ui::window_editor)
{
    ui->setupUi(this);
}

window_editor::~window_editor()
{
    delete ui;
}

File: window_editor.py
from PyQt5 import QtCore, QtGui, QtWidgets
import sys
from QtUi.window_editor import Ui_window_editor as UIM
import pygame
from Node.director import Director

pygame.init()



class NewMainWindow(UIM):
    def __init__(self):
        super(NewMainWindow, self).__init__()
        self.setupUi(MainWindow)

    def setup(self):
        pass


if __name__ == "__main__":
    app = QtWidgets.QApplication(sys.argv)
    MainWindow = QtWidgets.QMainWindow()
    ui = NewMainWindow()
    ui.setup()
    MainWindow.show()
    sys.exit(app.exec_())



